Makefile

```diff
@@ -140,19 +140,10 @@ install-tools:
 	cd $(TOOLS_MOD_DIR) && go install github.com/jcchavezs/porto/cmd/porto
 	cd $(TOOLS_MOD_DIR) && go install go.opentelemetry.io/build-tools/multimod
 
-.PHONY: otelcol
-otelcol:
-	go generate ./...
-	$(MAKE) build-binary-internal
-
-.PHONY: otelcol-unstable
-otelcol-unstable:
-	go generate ./...
-	$(MAKE) build-binary-internal-unstable
-
 .PHONY: run
-run:
-	GO111MODULE=on go run --race ./cmd/otelcol/... --config ${RUN_CONFIG} ${RUN_ARGS}
+run: build-binary-cmd-otelcol
```

Comment:
This currently fails with:

```
$ make run
make: *** No rule to make target 'build-binary-cmd-otelcol', needed by 'run'.  Stop.
```

Perhaps you need to rebase?

Reply:
For some reason even after rebasing the `build-binary-cmd-otelcol` rule was deleted in this PR; added it back in however and `make run` runs properly.